src/common/store-data-to-scan-node-result-converter.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { isNullOrUndefined } from '@microsoft/applicationinsights-core-js';
+import { isNullOrUndefined } from 'common/is-null-or-undefined';
```

Comment:
I still think it's more consistent with the rest of the codebase to just use `foo == null` instead of `isNullOrUndefined(foo)`. I would rather omit the helper and replace L97-104 with something like:

```ts
    if (
        (selectedTest == null) ||
        (assessmentStoreData?.assessments?.[selectedTest] == null)
    ) {​​
        return null;
    }​​
```

...and L62 with:

```ts
if (unifiedScanResultStoreData == null || results == null) {
```

If we want to discuss replacing `== null` with something that doesn't rely on the `== null` behavior matching `undefined`, I'd be okay with that as a more general change done consistently throughout the codebase (add a helper, use a codemod to update existing instances throughout, remove `{ null: 'ignore' }` from our `eqeqeq` eslint rule config), but I'd rather not create two conventions where there is currently one.

Reply:
Agreed that `== null` is the better approach at this time